Server/VikingVault.Services/TransactionService.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using VikingVault.DataAccess;
+using VikingVault.DataAccess.Models;
+using VikingVault.Services.Abstractions;
+
+namespace VikingVault.Services
+{
+    public class TransactionService : ITransactionService
+    {
+        private readonly VikingVaultDbContext _dbContext;
+
+        public TransactionService(VikingVaultDbContext context)
+        {
+            _dbContext = context;
+        }
+
+        public Transaction AddTransaction(Transaction tran)
+        {
+            throw new NotImplementedException();
+        }
+
+        public List<Transaction> GetTransactions(string userId)
+        {
+            int uid = int.Parse(userId);
+            return _dbContext.Transactions.Where(t => t.user.Id == uid).OrderByDescending(t => t.Date).Take(5).ToList();
```

Comment:
It's better to move these kind of variables outside (the `Take(5)` one), instead of writing them inline